lib/css/gluegun.css

```diff
@@ -32,8 +32,7 @@ h1 {
  * 2. Show the overflow in Edge and IE.
  */
 hr {
-  -webkit-box-sizing: content-box;
-          box-sizing: content-box;
+  box-sizing: content-box;
```

Comment:
I am not completely sure why the attributes for webkit disappeared while I generated the .css file, I am thinking because they weren't inside the .scss files to begin with. If necessary, I can change the code if there is a way to generate the CSS file with the webkit attributes.

Reply:
It should look good on all browsers. So whatever is needed.

Reply:
@kaankabalak Please use PostCSS to vendorize your CSS. 

https://github.com/postcss/autoprefixer